samples/dfu/updates/start_http_server.bash

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Require the node http-server to be installed
+# to do so, use npm i http-server
+http-server -p 8000
```

Comment:
This is an arbitrary requirement which isn't in documentation, there is no requirement for python3 for this, it should be available by default: `python3 -m http.server`

Also this isn't really of use for an nrf9160 without a tonne of further configuration, the system that runs this needs to be accessible to the LTE device, which if using a non-internal network operator means the PC must have port forwarding enabled to that machine via the internet.